programs/benchmark/Benchmark.cpp

```diff
@@ -572,6 +572,16 @@ class Benchmark : public Poco::Util::Application
 #pragma GCC optimize("-fno-var-tracking-assignments")
 #endif
 
+[[ maybe_unused ]] static std::string getHelpHeader()
+{
+    return
+        "Usage: clickhouse benchmark [options] < queries.txt\n"
+        "Usage: clickhouse benchmark [options] --query \"query text\"\n"
```

Comment:
It could be named as clickhouse-benchmark, and not necessarily located inside PATH, that's why `argv[0]` is better.

Reply:
Looked, that I already had 50% of apps detecting what to use(with dash or not). Fixed remaining.